### PR TITLE
Cocoapods development project intialization

### PR DESCRIPTION
**Pull request by eraydiler**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,4 @@
-MIT License
-
-Copyright (c) 2018 Hipo
+Copyright (c) 2018 eraydiler <eray.diler@gmail.com>
```
Comment (taylanpince) [inline LICENSE]: This should be Hipo

File: Magpie.podspec [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+#   * Think: What does it do? Why did you write it? What is the focus?
+#   * Try to keep it short, snappy and to the point.
+#   * Write the description between the DESC delimiters below.
+#   * Finally, don't worry about the indent, CocoaPods strips it!
+
+  s.description      = <<-DESC
+TODO: Add long description of the pod here.
+                       DESC
+
+  s.homepage         = 'https://github.com/eraydiler/Magpie'
```
Comment (taylanpince) [inline Magpie.podspec]: Wrong homepage

File: Magpie.podspec [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  # s.screenshots     = 'www.example.com/screenshots_1', 'www.example.com/screenshots_2'
+  s.license          = { :type => 'MIT', :file => 'LICENSE' }
+  s.author           = { 'eraydiler' => 'eray.diler@gmail.com' }
```
Comment (taylanpince) [inline Magpie.podspec]: Please use Hipo email

File: Magpie.podspec [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  s.source           = { :git => 'https://github.com/eraydiler/Magpie.git', :tag => s.version.to_s }
```
Comment (taylanpince) [inline Magpie.podspec]: Wrong source

File: README.md [hunk 5]
```diff
@@ -1,2 +1,206 @@
...
+)
+
+// HTTP body: {"foo": [1, 2, 3], "bar": {"baz": "qux"}}
+```
+
+Note that the `Content-Type` HTTP header field of an encoded request is set to `application/json`.
+
+## Author
+
+Hipo team
```
Comment (taylanpince) [inline README.md]: Please add your own name here Eray, you can also link your social media profile (i.e. Twitter) etc. if you wish

**Review by taylanpince — CHANGES_REQUESTED** [review]
Minor changes needed

**Comment by sakkaras** [comment]
@taylanpince I made a meeting with @eraydiler about the basic infrastructure and usage of the library. @eraydiler will write a prototype for what we discussed. Then you can give feedback over the prototype code.  

Is it ok for me to merge it since @eraydiler needs the project to work?

**Comment by taylanpince** [comment]
@sakkaras Sounds good!

**Review by taylanpince — APPROVED** [review]

**Pull request merged**